src/node-red/terminology/index.md

```diff
@@ -0,0 +1,89 @@
+---
+eleventyNavigation:
+  key: Terminology
```

Comment:
Is the `parent` added here magically? Or should that explicitly still be set? I *do* see the correct nav rendered, I just don't understand how it works. :)

Reply:
The `/node-red/node-red.json` adds the `learningResources` tag to all the content inside `/node-red/`, so they're all part of the same collection. If you add a key, the plugin will add it at the first level of the navigation of that collection, unless you assign a parent, then it would be nested under that parent in the navigation structure.